API.md

```diff
@@ -1,445 +1,648 @@
+try:
+    file_stat = os.stat('my-testfile')
+    file_data = open('my-testfile', 'rb')
+    minioClient.put_object('mybucket', 'myobject', file_data, file_stat.st_size)
+except ResponseError as err:
+    print(err)
+
+# Put a file with 'application/csv'.
+try:
+    file_stat = os.stat('my-testfile.csv')
+    file_data = open('my-testfile.csv', 'rb')
+    minioClient.put_object('mybucket', 'myobject.csv', file_data,
+                      file_stat.st_size, content_type='application/csv')
+except ResponseError as err:
+    print(err)
+```
+
+-------------------------------------
+<a name="fput_object">
+#### fput_object(bucket_name, object_name, file_path, content_type)
+Uploads contents from a file to objectName. 
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|---|---|---|
+|``bucket_name``   |*string*   |Name of the bucket.   |
+|``object_name``   |*string*    |Name of the object.   |
+|``file_path``   |*string*   |Path on the local filesystem to which the object data will be written. |
+|``content_type``   |*string* | Content type of the object. (optional, defaults to 'application/octet-stream').   |
+
+__Example__
+
+The maximum size of a single object is limited to 5TB. fput_object transparently uploads objects larger than 5MiB in multiple parts. This allows failed uploads to resume safely by only uploading the missing parts. Uploaded data is carefully verified using MD5SUM signatures.
+
+```py
+# Put an object 'myobject' with contents from '/tmp/otherobject'.
+try:
+    minioClient.fput_object('mybucket', 'myobject', '/tmp/otherobject')
+except ResponseError as err:
+    print(err)
+
+# Put on object 'myobject.csv' with contents from
+# '/tmp/otherobject.csv' as 'application/csv'.
+try:
+    minioClient.fput_object('mybucket', 'myobject.csv',
+                       '/tmp/otherobject.csv', content_type='application/csv')
+except ResponseError as err:
+    print(err)
+
+```
+------------------------------------
+<a name="stat_object">
+#### stat_object(bucket_name, object_name)
+Gets metadata of an object.
+
+__Parameters__
+
+|Param   |Type   |Description   |
+|---|---|---|
+|``bucket_name``   |*string*   |Name of the bucket.   |
+|``object_name``   |*string*    |Name of the object.   |
+
+__Return Value__
+
+<table>
+    <thead>
+        <tr>
+            <th>Param</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+           obj
+            </td>
+            <td> Object</td>
+            <td> object stat info for following format:
+            <ul>
+            <li>obj.size int: size of the object.</li>
+            <li>obj.etag string: etag of the object.</li>
+            <li>obj.content_type string: Content-Type of the object.</li>
+            <li>obj.last_modified time.time: modified time stamp.</li>
+            </ul>
+            </td>
+            </tr>
+               </tbody>
+</table>
+

```

Comment:
Param needs to be highlighted. Type needs to in italics. Object stat under description needs to be highlighted.